tools/sr-snapshots/Sources/SRSnapshotsCore/Utils/Hashing.swift

```diff
@@ -0,0 +1,20 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+import CommonCrypto
+
+internal protocol Hashing {
+    func hash(from data: Data) -> String
+}
+
+internal struct SHA256Hashing: Hashing {
```

Comment:
SHA256 for this is overkill along with taking a dependency on the external library, time taking.

SHA1 should work for our usecase as we want hash of given data.

Reply:
https://developer.apple.com/documentation/cryptokit/insecure/sha1

Reply:
Agree, we don't need secure hashing 👍 and SHA-1 could be better choice. I'll use it 👌

Reply:
Done ✅